package.json

```diff
@@ -85,7 +82,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "tslib": "^2.2.0",
-    "web-vitals": "^0.2.4"
+    "web-vitals": "^1.1.2"
```

Comment:
I think this one can go away because Heroic is not a website. But on the code side, might have some import that was added by create-react-app. So remove the file that uses it and remove this dependency.